### ISSUE TITLE
Convert Old Container Funcs to use UPID/Tasks

### ISSUE THREAD

**Issue by luthermonson** [luthermonson/go-proxmox#128]
The first Container funcs were created before UPID/Tasks were done so it used a ContainerStatus struct, this was changed to a string now but should ultimately be the UPID/Task system with a func profile looking similar to this...

```
func (c *Container) Start(ctx context.Context) (task *Task, err error)
func (c *Container) Stop(ctx context.Context) (task *Task, err error)
func (c *Container) Suspend(ctx context.Context) (task *Task, err error)
func (c *Container) Reboot(ctx context.Context) (task *Task, err error)
func (c *Container) Resume(ctx context.Context) (task *Task, err error)
func (c *Container) Shutdown(ctx context.Context, force bool, timeout int) (task *Task, err error)
```

**Comment by luthermonson** [comment]
@alperencelik for visibility

**Comment by luthermonson** [comment]
fixed by #129
